mantle/kola/tests/ignition/luks.go

```diff
@@ -176,6 +187,60 @@ func runTest(c cluster.TestCluster, tpm2 bool, threshold int, killTangAfterFirst
 	ut.LUKSSanityTest(c, tangd, m, tpm2, killTangAfterFirstBoot, rootPart)
 }
 
+func runCexTest(c cluster.TestCluster) {
```

Comment:
One thing I realized here is that we don't want this test to fail the whole run if we're running on an s390x machine that doesn't have a CEX device. (E.g. someone manually running in an IBM Cloud VM, or e.g. if the RHCOS devel pipeline used a different builder without it). I think here let's also check for `KOLA_CEX_UUID` and mark the test as skipped if it's missing (e.g. `c.Skip("No CEX device found in KOLA_CEX_UUID env var")`).

But note e.g. if they do `kola qemuexec --qemu-cex`, we'll still hard fail from the error in `AddCexDevice()`, which is what we want.